docker/helm/values.yaml

```diff
@@ -0,0 +1,82 @@
+# Default values for remote-shuffle-service.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# we provide a default rss image, you can also replace to your image
+image:
+  repository: registry.cn-shanghai.aliyuncs.com/emr-ack/remote-shuffle-service
```

Comment:
I think the public should not access this repo.

Reply:
changed to docker hub